pallets/time-release/src/weights.rs

```diff
@@ -58,106 +57,126 @@ pub trait WeightInfo {
 /// Weights for pallet_time_release using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	/// Storage: ParachainSystem ValidationData (r:1 w:0)
-	/// Proof Skipped: ParachainSystem ValidationData (max_values: Some(1), max_size: None, mode: Measured)
-	/// Storage: TimeRelease ReleaseSchedules (r:1 w:1)
-	/// Proof: TimeRelease ReleaseSchedules (max_values: None, max_size: Some(1449), added: 3924, mode: MaxEncodedLen)
+	/// Storage: `ParachainSystem::ValidationData` (r:1 w:0)
+	/// Proof: `ParachainSystem::ValidationData` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `TimeRelease::ReleaseSchedules` (r:1 w:1)
+	/// Proof: `TimeRelease::ReleaseSchedules` (`max_values`: None, `max_size`: Some(1449), added: 3924, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Locks` (r:1 w:1)
+	/// Proof: `Balances::Locks` (`max_values`: None, `max_size`: Some(1299), added: 3774, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Freezes` (r:1 w:0)
+	/// Proof: `Balances::Freezes` (`max_values`: None, `max_size`: Some(49), added: 2524, mode: `MaxEncodedLen`)
+	/// Storage: `System::Account` (r:1 w:1)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
 	fn transfer() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `113`
 		//  Estimated: `13824`
-		// Minimum execution time: 16_132_000 picoseconds.
-		Weight::from_parts(16_442_000, 13824)
-			.saturating_add(T::DbWeight::get().reads(2_u64))
-			.saturating_add(T::DbWeight::get().writes(1_u64))
+		// Minimum execution time: 32_133_000 picoseconds.
+		Weight::from_parts(32_860_000, 13824)
```

Comment:
nit: not sure what happened here that the weights doubled and even DB read and write is changed.

Reply:
This is a good catch. I am looking into it.

Reply:
Yeah, I looked into it. This seems more accurate. I could not find why the weights were generated otherwise.